version.txt

```diff
@@ -0,0 +1 @@
+v7.12.0-alpha.1
```

Comment:
I think you can remove the `v` before the version, and we can go directly with the version `7.12.0`.